### ISSUE TITLE
Laravel 10

### ISSUE THREAD

**Issue by albertobenavides** [devleaptech/laravel-h5p#1]
Is it possible to update this to work with Laravel 10?

**Comment by hareom284** [comment]
@albertobenavides  Yeah, I think no impact work with Laravel 10. Btw if you face any issues. You can open issues here.
